Notebooks/DC2Cookbook.ipynb

```diff
@@ -393,19 +393,35 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Make the `TemplateCoadd`s\n",
+    "### Make the `SkyMap` [issue [#121](https://github.com/LSSTDESC/DC2_Repo/issues/121)]\n",
+    "\n",
+    "The first thing we need to make is the sky map on which the coadded images will be projected for all bands. In order to ensure that the sky map will not depend on the list of input visits (their location on the sky or on the time and place the reprocessing happens), the sky map should be absolute (using `makeSkyMap.py` and the appropriate configuration) instead of relative to an input field (as done with `makeDiscreteSkyMap.py`). Making such a sky map will create thousands of tracts and patches, that will be fixed on the sky and usable in different reprocessing sessions. \n",
+    "\n",
+    "One possibility could be to use the [Rings sky map](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42), as done for the HSC data reprocessing with the LSST stack (and currently for CFHT data reprocessing as well). In order to make this sky map, some modifications of the `lsstSimMapper.py` [script](https://github.com/lsst/obs_lsstSim/blob/master/python/lsst/obs/lsstSim/lsstSimMapper.py) must be done to match what has been done in `hasMapper.py` ([here](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42) and [here](https://github.com/lsst/obs_subaru/blob/master/python/lsst/obs/hsc/hscMapper.py#L286)). A configuration file (`makeSkyMapConfig.py` below) is then needed to run `makeSkyMap.py`, containing information on how to parametrize the rings map. An example of such a configuration can be taken from the HSC reprocessing [repository](https://github.com/LSSTDESC/ReprocessingTaskForce/blob/master/config/w_2017_49/cfht/makeSkyMapConfig.py), which contains:\n",
+    "\n",
```

Comment:
I think here we need to cite the CFHTLS example but use the LSST values, eg `pixelScale = 0.20`. Who is best placed to advise on the values of `tractOverlap` and `numRings`? I think we can merge using your values as defaults, but we should target a quick question at the DM team to check unless you think the ones below _are_ what LSST would use.

Reply:
I gave this configuration file as an example, and we should definitely ask someone from DM to have a look at it and validate our choice of sky map. Maybe @TallJimbo or @PaulPrice might have a recommendation about that.

Reply:
After a short discussion about that on the slack `dm` channel with @TallJimbo, the only modification that must be done is for the `pixelScale`, as @drphilmarshall mentioned above. The number or rings could be decrease a little to make the tracts bigger and better match LSST, but that less important.

Reply:
You _might_ also want to adjust the `tractOverlap`. This sets the maximum size you can reliably extract from a single image. If you want to be able to extract NGC galaxies without having to warp tracts, you may want to make it bigger. If you only care about point sources and want to save disk space, you could make it smaller.